packages/compass-aggregations/src/components/stage-preview/stage-preview.jsx

```diff
@@ -52,6 +55,13 @@ class StagePreview extends Component {
     this.props.runOutStage(this.props.index);
   }
 
+  /**
+   * Called when the Atlas Signup CTA link is clicked.
+   */
+  onCreateAtlasCluster = () => {
+    this.props.openLink('https://www.mongodb.com/cloud/atlas/lp/general/try?utm_source=compass&utm_medium=product');
```

Comment:
(note: we need to wait for @mmarcon for the proper link here before merging this)

Reply:
I requested how to request it :D I'll know more once the US wakes up.